Makefile

```diff
@@ -87,7 +87,12 @@ endif
 	# For the specified GOOS + GOARCH, build all the binaries by default with CGO disabled
 	CGO_ENABLED=0 GOOS=${GOOS} GOARCH=${GOARCH} go install -trimpath $(EXTRA_BUILD_FLAGS) $(VT_GO_PARALLEL) -ldflags "$(shell tools/build_version_flags.sh)" ./go/...
 	# unset GOOS and embed local resources in the vttablet executable
-	(cd go/cmd/vttablet && unset GOOS && unset GOARCH && go run github.com/GeertJohan/go.rice/rice --verbose append --exec=$${HOME}/go/bin/${GOOS}_${GOARCH}/vttablet)
+	if [ -d /go/bin ]; then
```

Comment:
This can be made more reliable with `[ -f /run/.dockerenv ] || [ -f /run/.containerenv ]` (from some cursory googling

Reply:
Yeah, the comment here was to imply that we're inside the bootstrap image;  we don't really need to know if we're inside a container.  Will update the comment.